.github/workflows/reusable_test_wheels.yml

```diff
@@ -197,7 +201,7 @@ jobs:
         run: RUST_LOG=debug scripts/run_python_e2e_test.py --no-build # rerun-sdk is already built and installed
 
       - name: Run tests/roundtrips.py
-        if: ${{ inputs.PLATFORM != 'windows-x64' }}
+        if: ${{ inputs.PLATFORM != 'windows-x64' && !inputs.FAST }}
```

Comment:
would be good if you have any notes around on how/why these two in particular were slow it would be good to add

Reply:
I suspect it is the compiling and running of scores of examples. Unfortunately I cannot currently run them locally right now, partially because they relay on the `justfile` that is gone

Reply:
It would be good yes, but I fail to run them locally at the moment